lib/rouge/lexers/sas.rb

```diff
@@ -0,0 +1,568 @@
+# -*- coding: utf-8 -*- #
+
+module Rouge
+  module Lexers
+    class SAS < RegexLexer
+      title "SAS"
+      desc "SAS (Statistical Analysis Software)"
+      tag 'sas'
+      filenames '*.sas'
+      mimetypes *%w(
+        application/x-sas
+        application/x-stat-sas
+        application/x-sas-syntax
+      )
+
+      def self.data_step_statements
```

Comment:
This is more of a stylistic thing but for consistency, could you name the method with the same name as the instance variable? Ditto for the other methods that wrap reserved words.

Reply:
Thanks; done in https://github.com/rouge-ruby/rouge/pull/1107/commits/01c2cc4ec3f09e7ffb2f2ce7571f6324032d052f.